patches/Terraria/Terraria.Localization.Content.de-DE.Main.json

```diff
@@ -665,15 +665,15 @@
 		"Plead": "{0}s Bitte um den Tod wurde erhört.",
 		"Removed": "{0} wurde aus {1} entfernt.",
 		"Ripped": "{0}s Fleisch wurde von den Knochen gerissen.",
-		"Ruptured": "{0} lebensnotwendige Organe wurden zerfetzt.",
+		"Ruptured": "{0}s lebensnotwendige Organe wurden zerfetzt.",
 		"SkullCrushed": "{0}s Schädel wurde eingeschlagen",
 		"Slain": "{0} wurde ermordet.",
 		"Snapped": "{0} wurde in zwei Teile geteilt.",
 		"TornInHalf": "{0} wurde in zwei Hälften gerissen."
 	},
 	"DungeonDefenders2": {
 		"BartenderWarning": "Der Eternia-Kristall sträubt sich gegen dieses Gebiet und teleportiert sich sofort weg. Der Tavernenwirt hatte erwähnt, dass er in einem großen, flachen und offenen Gebiet aufgestellt werden sollte ...",
-		"CantSummonTower": "It doesn't seem to work without an Etheria Crystal nearby...",
+		"CantSummonTower": "Es sieht nicht aus ohne einen Eternia-Kristall in der nähe zu funktionieren...",
```

Comment:
I would write it like `Es scheint nicht ohne einen Eternia-Kristall in der nähe zu funktionieren...`